libs/coin-modules/coin-xrp/src/signer.ts

```diff
@@ -0,0 +1,17 @@
+export type XrpAddress = {
+  publicKey: string;
+  address: string;
+  chainCode?: string;
+};
+
+export type XrpSignature = string; // `0x${string}`
```

Comment:
What do you mean by enforce the type ? Using `0x${string}` instead of `string` ?
If so, unfortunately, I really like this templated type, but it requires to do a lot of assertion everywhere if your base layer is not implemented straight away with this in mind. Doing an ``` "0xabc" as `0x${string}` ``` makes it kinda useless for now. 
But yeah, if possible it would be nice to use it in the whole code base and implement it at every level of LL

Reply:
Yeap, we could later use it, but from the bottom up, to be sure that there is no mistake.